src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -319,25 +331,31 @@ def _validate_ensemble_config(cls, ensemble_config, config_path):
     def read_forward_model(
         cls, installed_jobs, substitution_list, config_dict, config_file
     ):
+        all_errors = []
         jobs = []
-        for job in config_dict.get(ConfigKeys.FORWARD_MODEL, []):
-            if len(job) > 1:
-                unsubstituted_job_name, args = job
+        for job_as_list in config_dict.get(ConfigKeys.FORWARD_MODEL, []):
+            if len(job_as_list) > 1:
+                unsubstituted_job_name, args = job_as_list
             else:
-                unsubstituted_job_name = job[0]
+                unsubstituted_job_name = job_as_list[0]
                 args = []
             job_name = substitution_list.substitute(unsubstituted_job_name)
+
+            job = None
             try:
                 job = copy.deepcopy(installed_jobs[job_name])
-            except KeyError as err:
-                raise ConfigValidationError(
-                    errors=(
-                        f"Could not find job {job_name!r} in list of installed jobs: "
-                        f"{list(installed_jobs.keys())!r}"
-                    ),
-                    config_file=config_file,
-                ) from err
-            if args:
+            except KeyError:
+                all_errors.append(
+                    ConfigValidationError(
+                        errors=(
+                            f"Could not find job {job_name!r} in list of installed "
+                            f"jobs: "
+                            f"{list(installed_jobs.keys())!r}"
+                        ),
+                        config_file=config_file,
+                    )
+                )
```

Comment:
Perhaps it would simplify the logic a bit to use a continue here instead of relying on the condition `job is not None` ?

Reply:
Done